lib/conjur/variable.rb

```diff
@@ -205,5 +205,19 @@ def value(version = nil)
       url << "?version=#{version}" if version
       self[url].get.body
     end
+
+    # Set the expiration for the variable.
+    #
+    # You must have the **`'update'`** permission on a variable to call this method.
+    #
+    # @example Set the expiration for a variable to tomorrow
+    #   var = api.variable 'my-secret'
+    #   var.expire 'P1D'
+    # @param duration ISO8601 duration the variable should be valid
```

Comment:
As commented above: perhaps it should take a Ruby-ish representation instead of forcing the user to convert.